src/main/java/io/swagger/repository/MediaRepository.java

```diff
@@ -0,0 +1,11 @@
+package io.swagger.repository;
+
+import io.swagger.entity.media.MediaInfo;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.Optional;
+
+public interface MediaRepository extends CrudRepository<MediaInfo, String> {
+    Optional<MediaInfo> findByTitle(String title);
```

Comment:
I originally planned to have the MediaCache simply be a Java object.... but a database repository makes sense for simplicity. The service code checks the database first... if no data stored locally, make a remote call (not yet implemented).